rustls/src/record_layer.rs

```diff
@@ -62,12 +62,12 @@ impl RecordLayer {
     /// an error is returned.
     pub(crate) fn decrypt_incoming(
         &mut self,
-        encr: OpaqueMessage,
+        encr: BorrowedOpaqueMessage,
```

Comment:
Nit: let's make the lifetime explicit?

Reply:
Done. A later commit further refines this to a named lifetime.